modules/cco-ccoctl-deleting-sts-resources.adoc

```diff
@@ -82,21 +82,23 @@ $ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}'
 +
 [source,terminal]
 ----
-$ oc adm release extract --credentials-requests \
+$ oc adm release extract
+  --credentials-requests \
   --cloud=gcp \
-  --to=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
+  --to=<path_to_directory_with_list_of_credentials_requests>/credrequests \// <1>
```

Comment:
"ccoctl gcp delete" needs the CR list so I think "--included" is needed here too, with kubeconfig instead of install-config.

Reply:
good catch, thanks!